tap_linkedin_ads/client.py

```diff
@@ -63,16 +63,17 @@ def get_next_page_token(
         # If pagination is required, return a token which can be used to get the
         #       next page. If this is the final page, return "None" to end the
         #       pagination loop.
-
         resp_json = response.json()
         if previous_token is None:
             previous_token = 0
-
-        elements = resp_json.get("elements")
-
-        if len(elements) == 0 or len(elements) == previous_token + 1:
-            return None
-
+        try:
+            elements = resp_json.get("elements")
+            if len(elements) == 0 or len(elements) == previous_token + 1:
```

Comment:
```suggestion
            if elements and len(elements) == 0 or len(elements) == previous_token + 1:
```

@NeilGorman104 I dont know exactly what the data looks like but this feels like it can be refactored to instead use a cleaner check for the existence for the `elements` key in the resp_json  or defaulting it to an empty list because I think the Exception would be raised by len(None) when elements doesnt exist in the dict. Then you shouldnt need the try/except anymore.

Reply:
Hey Pat, we have resolved this issue and remove the try/except in favor of an if/else in our latest PR